pkg/daemon/update.go

```diff
@@ -782,14 +808,14 @@ func (dn *Daemon) updateFiles(oldConfig, newConfig *mcfgv1.MachineConfig) error
 // all the files, units that are present in the old config but not in the new one.
 // this function will error out if it fails to delete a file (with the exception
 // of simply warning if the error is ENOENT since that's the desired state).
-func (dn *Daemon) deleteStaleData(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+func (dn *Daemon) deleteStaleData(oldConfig, newConfig *igntypes.Config) error {
```

Comment:
Minor nit, to keep it consistent I think we should refer "Config" as MachineConfig and "IgnConfig" as ignition config. For example here you changed oldConfig from a MC to a ign config, but didn't change the naming, which is slightly confusing